package.json

```diff
@@ -60,11 +60,10 @@
   "dependencies": {
     "@tensorflow/tfjs": "^1.5.1",
     "long": "^4.0.0",
-    "syft-proto": "git+https://github.com/OpenMined/syft-proto.git",
-    "tuple-w": "^1.1.1"
+    "syft-proto": "git+https://github.com/OpenMined/syft-proto.git"
   },
   "peerDependencies": {
-    "@tensorflow/tfjs": "^1.2.5"
+    "@tensorflow/tfjs": "^1.5.1"
```

Comment:
We should leave the peer dependency as low as possible. This is the minimum version of TFJS that a user is allowed to use with syft.js.  We should leave the value where it is or consider lowering it further.

Reply:
Lowered in #98